CHANGELOG.md

```diff
@@ -8,37 +8,92 @@ Changelog
   * acl (@saqibraza)
   * ace (@yjyongz)
   * remark ace (@bansalpradeep)
+  * **Platforms:** Nexus (3|9)k
 * EVPN
   * evpn_vni (@andish)
+  * **Platforms:** Nexus (3|9)k
```

Comment:
Yes, I started with that but then did not like how it looked.  If you feel strongly about this I can change it back because I don't have a strong opinion either way.